recipe/bld.bat

```diff
@@ -0,0 +1,14 @@
+mkdir build
+cd build
+
+cmake -G "NMake Makefiles" ^
+      -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+      -D CMAKE_BUILD_TYPE=Release ^
+      ..
+if errorlevel 1 exit 1
+
+nmake
+if errorlevel 1 exit 1
+
```

Comment:
Again this was more of a note of what we try to do on Windows.

Though I went back to see what we decided with the *NIXes and it looks like we decided not to do the testing because it started pulling in `numpy` and things. There were some test failures too. All of this would require a fair bit of patching, so it was skipped. I still agree with that decision. So, short answer, don't worry about it. :smile: